src/features/footer/MobileFooter.jsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { sendTrackEvent } from '@edx/frontend-platform/analytics';
+import { ensureConfig } from '@edx/frontend-platform/config';
+import { AppContext } from '@edx/frontend-platform/react';
+
+ensureConfig([
+  'ADA_URL',
+  'LMS_BASE_URL',
+  'LOGO_TRADEMARK_URL',
+  'PRIVACY_POLICY_URL',
+  'TERMS_OF_SERVICE_URL',
+  'COPY_RIGHT_STRING',
+], 'DesktopFooter component');
+
+const EVENT_NAMES = {
```

Comment:
I see that this is the same we have on DesktopFooter, why not defining this in another place and just importing it here and on DesktopFooter?

Reply:
I have moved ensure config to the parent (Footer.jsx) for both elements to utilize.

Reply:
that's fine, however I was referring to the EVENT_NAMES variable, I think it can be defined just once and exported, instead of having it on both MobileFooter and DesktopFooter